packages/nvidia_nat_test/src/nat/test/llm.py

```
@@ -203,3 +203,31 @@ async def ainvoke_stream(self, messages: Any | None = None, **_kwargs: Any) -> A
             yield chooser.next_response()
 
     yield AgnoTestLLM()
+
+
+@register_llm_client(config_type=TestLLMConfig, wrapper_type=LLMFrameworkEnum.ADK)
+async def test_llm_adk(config: TestLLMConfig, builder: Builder):
```

Ruff (0.13.3)

Unused function argument: `builder`

(ARG001)